x-pack/plugins/observability_solution/apm/public/components/app/dependencies_inventory/random_sampler_badge.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+
+export function RandomSamplerBadge() {
+  return (
+    <>
+      <EuiToolTip
+        content={i18n.translate('xpack.apm.dependencies.randomSampler.tooltip', {
+          defaultMessage:
+            'This view is using random sampler aggregations due to the amount of data found in the selected time range. Sampling has been configured to balance accuracy and speed. Reduce the time range to see un-sampled data.',
+        })}
+      >
+        <EuiBadge iconType="iInCircle" color="hollow">
+          {i18n.translate('xpack.apm.dependencies.randomSampler.badge', {
+            defaultMessage: `Based on sampled transactions`,
```

Comment:
This is re-using the message that is displayed already when falling back from metrics to transactions. I can't tell if that's good or bad. It could be slightly confusing when debugging this because it's not immediately obvious if the message refers to sampled spans or sampled transactions.

Reply:
sounds good! I'll change it.